react/features/base/dialog/components/SimpleBottomSheet.native.js

```diff
@@ -0,0 +1,189 @@
+// @flow
+
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import {
+    Modal,
+    Text,
+    TouchableHighlight,
+    TouchableWithoutFeedback,
+    View
+} from 'react-native';
+import { connect } from 'react-redux';
+
+import { hideDialog } from '../actions';
+import { Icon } from '../../font-icons';
+
+import { bottomSheet as styles } from './styles';
+
+/**
+ * Underlay color for the buttons on the sheet.
+ *
+ * @type {string}
+ */
+const BUTTON_UNDERLAY_COLOR = '#eee';
+
+/**
+ * A component emulating Android's BottomSheet, in a simplified form.
+ * It supports text options with an icon, which the user can tap. The style has
+ * been implemented following the Material Design guidelines for bottom
+ * sheets: https://material.io/guidelines/components/bottom-sheets.html
+ *
+ * For all intents and purposes, this component has been designed to work and
+ * behave as a {@code Dialog}.
+ */
+class SimpleBottomSheet extends Component<*> {
+    /**
+     * {@code SimpleBottomSheet}'s React {@code Component} prop types.
+     *
+     * @static
+     */
+    static propTypes = {
+        /**
+         * Used to hide the sheet on cancel.
+         */
+        dispatch: PropTypes.func,
+
+        /**
+         * Handler for the cancel event, which happens when the user dismisses
+         * the sheet.
+         */
+        onCancel: PropTypes.func,
+
+        /**
+         * Handler for the event when an option has been selected in the sheet.
+         */
+        onSubmit: PropTypes.func,
+
+        /**
+         *
+         */
+        options: PropTypes.arrayOf(PropTypes.shape({
+            iconName: PropTypes.string,
+            selected: PropTypes.bool,
+            text: PropTypes.string
```

Comment:
Where are these documented with jsdocs? If they're not documented anywhere, please document them.